src/Controller/TwigAwareController.php

```diff
@@ -87,21 +78,4 @@ protected function renderTemplate($template, array $parameters = [], ?Response $
         // Process the snippet Queue on the Response
         return $this->widgets->processQueue($response);
```

Comment:
move to onKernelResponse

Reply:
Added a ProcessWidgetsQueueSubscriber, removed any mention of widgets from TwigAwareController.